ionic_langchain/prompt.py

```diff
@@ -1,7 +1,7 @@
 TOOL_PROMPT = """
```

Comment:
This could stand for a broader update this is pretty old. Added a ticket for later: https://linear.app/ionic-commerce/issue/ION-535/update-langchain-prompt